src/app/public/modules/selection-box/fixtures/selection-box.component.fixture.ts

```diff
@@ -49,4 +62,23 @@ export class SelectionBoxTestComponent {
       name: 'blue'
     }
   ];
+
+  constructor(
+    private formBuilder: FormBuilder
+  ) {}
+
+  public ngOnInit(): void {
+    this.myForm = this.formBuilder.group({
+      checkboxes: this.buildCheckboxes()
+    });
+
+    // this.myForm.get('checkboxes').get('1').disable();
```

Comment:
Leftover comment?

Reply:
Thanks for catching that; fixed in last commit